tests/coreos-install/util/util.go

```diff
@@ -106,14 +115,43 @@ func FetchLocalImage(t *testing.T) string {
 	return tmpDir
 }
 
-func DownloadFile(tmpDir, name string) error {
+// Used to get defaults for channel, board, & version, first checks if the
+// host machine is Container Linux and if so uses the data from the machine
+// otherwise defaults to stable, amd64-usr, & current respectively
+func GetDefaults(t *testing.T) (string, string, string) {
+	data, err := ioutil.ReadFile("/usr/lib/os-release")
+	if err != nil {
+		return "stable", "amd64-usr", "current"
+	}
+
+	os, err := regexpSearch("id", "ID=(.*)", data)
```

Comment:
In the examples present on `os-release(5)` the `ID=` field seems like it has special rules. Does the quote checking not apply here?

Reply:
The second paragraph of the Description section says when quotes are required, but doesn't say whether they're optional or forbidden otherwise.